core/contracts/oracle/implementation/DesignatedVoting.sol

```diff
@@ -19,7 +19,7 @@ contract DesignatedVoting is Withdrawable {
      ****************************************/
 
     enum Roles {
-        Owner, // Can set the Voter role. Is also permanently permissioned as the minter roll.
```

Comment:
aaahhh yes. the Owner roll
![french-bread-roll1-480x360](https://user-images.githubusercontent.com/12886084/80578743-437fa900-8a09-11ea-9f04-4d0a231a0bc9.jpg)